Libraries/LibDNS/Resolver.h

```diff
@@ -69,7 +104,10 @@ class LookupResult : public AtomicRefCounted<LookupResult>
     void add_record(Messages::ResourceRecord record)
     {
         m_valid = true;
-        auto expiration = record.ttl > 0 ? Optional<Core::DateTime>(Core::DateTime::from_timestamp(Core::DateTime::now().timestamp() + record.ttl)) : OptionalNone();
+        auto expiration = record.ttl > 0
+            ? Optional<Core::DateTime>(
+                  Core::DateTime::from_timestamp(Core::DateTime::now().timestamp() + record.ttl))
+            : OptionalNone();
```

Comment:
:shrug: if you restore this line, does clang-format-20 really format it this way? If it does I'll stop complaining, but it sure looks weird.

Reply:
yeah it didn't change it at all, I wonder if clion was doing something weird with it?
I for sure wouldn't format things that way myself...